xarray/backends/api.py

```diff
@@ -444,6 +448,11 @@ def open_dataset(
           appropriate locks are chosen to safely read and write files with the
           currently active dask scheduler. Supported by "netcdf4", "h5netcdf",
           "scipy", "pynio", "pseudonetcdf", "cfgrib".
+        - 'inline_array': How to include the array in the dask task graph. By
```

Comment:
This applies to all engines, and so should be an explicit kwarg in the signature?

Reply:
I put it here because `lock` is also here, and `lock` is passed down to `dask.array.from_array` in a similar way. I can move it though?

Reply:
I think so but this stuff is all very fuzzy to me :)

Reply:
moved now